app/controllers/trading_partners_controller.rb

```diff
@@ -15,6 +15,7 @@ def update
     if @trading_partner.valid?
       redirect_to goods_nomenclature_path(
         country: @trading_partner.country,
+        anchor: trading_partner_params['anchor'],
```

Comment:
Shouldn't this be part of the goods_nomenclature_path construction? Is the referer blank for some reason?

```ruby
  def anchor
    referer&.fragment
  end
```

Reply:
anchor is provided by Rails

Reply:
So why can't we use that?